nrm_django/grants/models.py

```diff
@@ -414,7 +421,7 @@ class Grant(models.Model):
         max_length=34,
         blank=True,
         null=True,
-        help_text="Cooperators agreement number, if different then the Forest Service Agreement Number",
+        help_text="Cooperator's agreement number, if different then the Forest Service Agreement Number",
```

Comment:
s/then/than/

Reply:
Nice catch. I spotted the apostrophe but not then/than. Also, don't blindly cut/paste help text from the user guide.